node_modules/oae-admin/tenants/tenants.html

```diff
@@ -112,6 +122,12 @@
                             </div>
                         </div>
                         <div class="form-group">
+                            <label for="tenants-create-email-domain" class="col-xs-12 col-sm-3 control-label">__MSG__EMAIL_DOMAIN_COLON__</label>
+                            <div class="col-xs-12 col-sm-9">
+                                <input type="text" name="emailDomain" id="tenants-create-email-domain" placeholder="__MSG__EMAIL_DOMAIN__" class="form-control required url"/>
```

Comment:
Yeah, you can hold off here for now. I'll be reviewing the tenant paging follow-up later today